### PR TITLE
Add more WebAssembly feature extensions

### PR DESCRIPTION
**Pull request by dschuff**
This adds the data for WebAssembly extensions including
mutable global import/export, sign extension operators, threads/atomics, bulk
memory operations, nontrapping float-to-int conversion, bigint integration,
multi-value and SIMD. These are features that have been shipped in all major
browser engines.

### CODE REVIEW

**Comment by Fyrd** [comment]
Thanks! I'll merge this PR now and review support later.

**Comment by Fyrd** [comment]
@dschuff According to your data in this PR multiple browsers support threads/atomics, but looking at https://wasm-feature-detect.surma.technology/ across even current browsers I'm seeing the test failing every time.

**Comment by dschuff** [comment]
Using wasm threads (and SharedArrayBuffer) requires the page to be cross-origin isolated with COOP/COEP, so if it's not served the right way, threads won't work. Most likely the problem is that that demo URL isn't being served right (it can actually quite a pain for users, depending on how their site is architected). The corresponding page at https://webassembly.org/features/ (which just embeds the same wasm-feature-detect library) is set up correctly to serve with COEP, and you should hopefully be able see that it works.

**Comment by Fyrd** [comment]
Ah, thanks, you're right!

The next thing I notice is that you have Opera versions set as if their version matches the Chromium version which is not the case. Instead it's 14 points off (as of Opera 69, 13 points off for Opera 68 and below), so if it's supported in Chrome 85 it should be in Opera 71.

**Comment by dschuff** [comment]
Oops, I didn't realize that. I can open another PR to fix that if you want.

File: features-json/wasm-bigint.json [hunk 1]
```diff
@@ -0,0 +1,603 @@
+{
+  "title":"WebAssembly Non-trapping float-to-int Conversion",
```
Comment (Fyrd) [inline features-json/wasm-bigint.json:2]: @dschuff I noticed this is the wrong title, have replaced it with "WebAssembly BigInt to i64 conversion in JS API"

Comment (dschuff) [reply features-json/wasm-bigint.json:2]: Ah thanks. I'm sure I just forgot to update it when copy/pasting.

**Comment by dschuff** [comment]
> Alright, all of these now appear on the site.

Thanks for your help!

**Pull request merged**